platform/darwin/src/MGLGeometry_Private.h

```diff
@@ -51,6 +51,7 @@ NS_INLINE mbgl::EdgeInsets MGLEdgeInsetsFromNSEdgeInsets(NSEdgeInsets insets) {
     @param latitude The latitude of the point at the center of the viewport.
     @param size The size of the viewport.
     @return An altitude measured in meters. */
+MGL_EXTERN
```

Comment:
Do we really need to `extern` symbols in headers with private or project visibility? What about symbols declared in .m files, then?